lib/ansible/plugins/inventory/script.py

```diff
@@ -187,7 +187,16 @@ def get_host_variables(self, path, host):
             sp = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
         except OSError as e:
             raise AnsibleError("problem running %s (%s)" % (' '.join(cmd), e))
-        (out, err) = sp.communicate()
+        (out, stderr) = sp.communicate()
+
+        err = to_native(stderr or "")
```

Comment:
New controller code should use `to_text` instead of `to_native`, since the controller only runs on Python 3.

You'll see a lot of existing code using `to_native`, which can be left as-is for now.

Reply:
do we even need `to_X ` at this point? won't `str()` 'just work'TM?

Reply:
Do I have to change all to_native to to_text or just the one that I added?

Reply:
just the one added

Reply:
to_text() or str()?

Reply:
use to_text for now, that question is probably something for a core meeting

Reply:
to_text has failed, may I have to import a lib.

Reply:
it is imported in L40, same as to_native